Makefile

```
@@ -20,7 +20,13 @@ CI_FLAGS =\
 	DBT_LOG_FORMAT=json
 
 .PHONY: dev
-dev: ## Installs dbt-* packages in develop mode along with development dependencies.
+dev: ## Installs dbt-* packages in develop mode along with development dependencies and pre-commit.
+	@\
+	pip install -r dev-requirements.txt -r editable-requirements.txt && \
+	pre-commit install
```

I'm not a Make guru, but it seems like this would be more DRY if re-factored like this:
```suggestion
dev: dev_req ## Installs dbt-* packages in develop mode along with development dependencies and pre-commit.
	@\
	pre-commit install
```

---

i like this - thanks for the tip! hadn't considered this but makes total sense. it should now be written this way in the latest commit (b4bf3b4)